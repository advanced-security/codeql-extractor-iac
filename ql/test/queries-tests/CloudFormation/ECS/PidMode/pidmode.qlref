security/CloudFormation/ECS/PidMode.ql
