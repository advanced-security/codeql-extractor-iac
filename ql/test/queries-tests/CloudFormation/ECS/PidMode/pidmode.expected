| ecs.yml:9:5:40:2 | CloudFormation ECS Task Definition | PidMode should be "task" for ECS tasks |